src/PrestaShopBundle/Controller/Admin/Sell/Catalog/CategoryController.php

```diff
@@ -0,0 +1,231 @@
+
+        return $this->render('@PrestaShop/Admin/Sell/Catalog/Categories/categories.html.twig', [
+            'categoriesGrid' => $this->presentGrid($categoryGrid),
+            'categoriesKpi' => $categoriesKpiFactory->build(),
+            'layoutHeaderToolbarBtn' => $this->getCategoryToolbarButtons($request),
+            'currentCategoryView' => $categoryViewData,
+        ]);
+    }
+
+    /**
+     * Toggle category status.
+     *
+     * @param int $categoryId
+     *
+     * @return RedirectResponse
+     */
+    public function processStatusToggleAction($categoryId)
+    {
+        try {
+            $command = new ToggleCategoryStatusCommand(new CategoryId($categoryId));
+
+            $this->getCommandBus()->handle($command);
+
+            $this->addFlash(
+                'success',
+                $this->trans('The status has been updated successfully', 'Admin.Notifications.Success')
+            );
+        } catch (CategoryException $e) {
+            $this->addFlash('error', $this->handleUpdateStatusException($e));
+        }
+
+        return $this->redirectToRoute('admin_category_listing');
+    }
+
+    /**
+     * Process bulk action for categories status enabling.
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function processBulkStatusEnableAction(Request $request)
+    {
+        $this->updateBulkStatus($request->request->get('categories_bulk'), CategoryStatus::ENABLED);
+
+        return $this->redirectToRoute('admin_category_listing');
+    }
+
+    /**
+     * Process bulk action for categories status disabling.
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function processBulkStatusDisableAction(Request $request)
+    {
+        $this->updateBulkStatus($request->request->get('categories_bulk'), CategoryStatus::DISABLED);
+
+        return $this->redirectToRoute('admin_category_listing');
+    }
+
+    /**
+     * Show category for editing.
+     *
+     * @param int $categoryId
+     *
+     * @return Response
+     */
+    public function editAction($categoryId)
+    {
+        return $this->redirect(
+            $this->getAdminLink('AdminCategories', [
+                'id_category' => $categoryId,
+                'updatecategory' => 1,
+            ])
+        );
+    }
+
+    /**
+     * Update categories status.
+     *
+     * @param int[] $categoryIds
+     * @param string $newStatus
+     */
+    protected function updateBulkStatus(array $categoryIds, $newStatus)
+    {
+        try {
+            $command = new UpdateCategoriesStatusCommand(
+                $categoryIds,
+                new CategoryStatus($newStatus)
+            );
+
+            $this->getCommandBus()->handle($command);
+
+            $this->addFlash(
+                'success',
+                $this->trans('The status has been successfully updated.', 'Admin.Notifications.Success')
```

Comment:
Why is there two translation keys with the same meaning and not the same wording? Previously, there was _The status has been updated successfully_ (Admin.Notifications.Success). Now I can read _The status has been successfully updated._ (Admin.Notifications.Success).

Reply:
i dont really know, by default, when you change category status, you dont get any confirmation message in legacy page, so i picked the most relevant message i could find. is it wrong? :thinking:

Reply:
Okay, @sarjon, no worries, both are right! But for harmonization purpose, I'd prefer you pick up one wording and use it through this PR. If you don't mind. Here, you can check lines 90 and 162, they contain a (slightly) different wording when they mean the exact same thing. I think it is better to use the same translation key for the two lines.

Reply:
oh yes, you are right, ill update that. :+1:

Reply:
Thank you very much, Šarūnas!